modules/users/client/views/authentication/signup.client.view.html

```diff
@@ -24,7 +24,7 @@ <h3>Join us!</h3>
             <div class="signup-step-indicator">1</div>
             Info
           </div>
-          <div class="signup-step-line"></div>
+          <!-- <div class="signup-step-line"></div>
```

Comment:
You don't need to be scared of deleting stuff, that's why we have version control :)

Reply:
Yes, it's better that it's gone so that what's running is clean. If you ever want to see old code you can `git checkout <commit number>`. `git stash` is also your friend in the case that you have WIP that you didn't want to commit.